src/framework/ui/iuiactionsregister.h

```diff
@@ -38,10 +38,13 @@ class IUiActionsRegister : MODULE_EXPORT_INTERFACE
 
     virtual void reg(const IUiActionsModulePtr& actions) = 0;
 
-    virtual const UiAction& action(const muse::actions::ActionCode& code) const = 0;
-    virtual const std::vector<UiAction> getActions() const = 0;
-    virtual UiActionState actionState(const muse::actions::ActionCode& code) const = 0;
-    virtual async::Channel<muse::actions::ActionCodeList> actionStateChanged() const = 0;
+    virtual const std::vector<UiAction> actionList() const = 0;
```

Comment:
Why a `const` return value if it's not a reference?

Reply:
removed const 
(I don't know why the author added const, probably wanted to somehow semantically show that there is no need to change this list...
I just renamed the method to match our style)